company/strategy.md

```diff
@@ -1,36 +1,129 @@
-# Sourcegraph strategy
+# Sourcegraph master plan
+
+Everything we do stems from our [purpose](#purpose), to help *everyone* build software. We'll accomplish that over the next 30 years by following our strategy:
+
+- 30 years
+  - [Purpose](#purpose)
+  - [Mission](#mission)
+  - [Values](#values)
+- 10 years
+  - [10-year vision](#10-year-vision)
+  - [Big Code](#big-code)
+- 3 years
+  - [3-year vision](#3-year-vision)
+  - [Principles](#principles)
+  - [Assumptions](#assumptions)
+- 1 year
+  - [1-year vision](#1-year-vision)
+  - [Product direction](#product-direction)
+  - [OKRs](#okrs)
 
 ## Purpose
 
-Sourcegraph exists to create a world where everyone builds software. A world where everyone, not just ~0.2% of the world's population (people who can code today), builds software will see faster and more broadly beneficial technological progress.
+We want everyone to be able to build software. A world where everyone, not just the ~0.2% who can code today, builds software will see faster and more broadly beneficial technological progress.
```

Comment:
I know this has been up for awhile, but is this really our purpose? What immediately comes to mind when reading this purpose is some sort of learning tool that teaches non-developers the fundamentals of programming. That is not what we do. What we do is understand the universe of code and put that at every developer's fingertips. This is useful to both experienced developers and newbies, but doesn't feel the same as enabling "everyone to be able to build software".

Alternatives:
* We want everyone to be able to participate in the building of software.
* We want to make software development effective at scale.
* More and more people are going to build software, and Sourcegraph is the tool the world needs to handle the influx of new developers and code.
* We want to make the universe of code accessible to every developer, regardless of where they are in the world, what company they work for, and their experience level.

Reply:
I don't think our goal is to help everyone write software, for the same reasons Beyang mentions. But there is a need for many people to understand complex and large codebases. What is the system of record for VP Engineerings?